src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -111,6 +111,52 @@
         [expand-button expanded? chat-id message-id])])
    {:justify-timestamp? true}])
 
+(defview contact-request-message
+  [{:keys [chat-id message-id content from timestamp-str group-chat outgoing current-public-key expanded?] :as message}]
+  (letsubs [{:keys [last-updated
+                    pending?]} [:contacts/contact-by-identity chat-id]]
+    (let [sent-by-us? (= current-public-key from)
+          pending-state? (if sent-by-us?
+                           ;; last-updated is only updated if we have been added to their contacts
+                           (not (pos? last-updated))
```

Comment:
`not pos?` as in `zero?` or could it be `nil`?

Reply:
it could be `nil` for now, but it is coherced to int from realm when loading from it, but I could just change that